examples/basic-sc/README.md

```diff
@@ -180,18 +140,32 @@ registry:
   password: a-very-hard-to-break-password
 ```
 
+Next the developer needs to give Cartographer permission to create the
+items that will be stamped out in the supply chain. That will be done
+with a service account, whose name will be referenced in the workload.
+You can update [`values.yaml`](./values.yaml) with the desired name:
```

Comment:
Also it would be beneficial to make it a imperative statement : ` Update values.yaml with your registry details`

Reply:
I updated this with imperative language from your suggestion, but then read the comment below noting that no change is needed to this value, so I've reverted it.